vms/platformvm/txs/executor/staker_tx_verification.go

```diff
@@ -686,11 +691,15 @@ func verifyAddPermissionlessDelegatorTx(
 		return err
 	}
 
-	canDelegate, err := canDelegate(chainState, validator, maximumWeight, newStaker)
+	fitsWindow := fitsValidationWindow(validator, newStaker)
```

Comment:
Is it possible to add a unit test to verify this behavior now?

Reply:
it was considered in current unit tests, but they were also expecting the over delegation message

Reply:
if we reuse txs.BoundedBy function for delegators, we got the unit tests already